extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -46,6 +46,7 @@ const TAG_ = 'amp-iframe';
 
 /** @const {!Array<string>} */
 const ATTRIBUTES_TO_PROPAGATE = [
+  'title',
```

Comment:
Nit: The rest of these appear to be alphabetised. Might want to keep the pattern up.